.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,105 @@
+name: Bug report
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **THIS IS NOT A SUPPORT CHANNEL!**
+        **IF YOU HAVE SUPPORT QUESTIONS ABOUT RUNNING OR CONFIGURING YOUR OWN HOME SERVER**,
+        please ask in **[#synapse:matrix.org](https://matrix.to/#/#synapse:matrix.org)** (using a matrix.org account if necessary).
+
+        If you want to report a security issue, please see https://matrix.org/security-disclosure-policy/
+
+        This is a bug report form. By following the instructions below and
+        completing the sections with your information, you will help the us to get all
+        the necessary data to fix your issue.
+
+        You can also preview your report before submitting it.
```

Comment:
Are the line breaks mid-sentence going to show in the template? They do on https://github.com/matrix-org/synapse/blob/rei/bug_report_form/.github/ISSUE_TEMPLATE/BUG_REPORT.yml and it looks kinda weird, so if so I'd probably be in favour of removing them.